hack/manifest-gen/assets/charts/eck/templates/_helpers.tpl

```diff
@@ -94,6 +94,14 @@ RBAC permissions
   - update
   - patch
   - delete
+- apiGroups:
+  - storage.k8s.io
+  resources:
+  - storageclasses
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
Hm now that I think about it again: since storage classes are a cluster-wide resource, they need to live in a ClusterRole with a ClusterRoleBinding, not in a Role with a RoleBinding?

Reply:
https://github.com/elastic/cloud-on-k8s/pull/3752#issuecomment-694339216